### PR TITLE
hw03 solution

### PR DESCRIPTION
**Pull request by Vlad311010**

### CODE REVIEW

File: hw02/hw02.sln [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿
```
Comment (IgorKheidorov) [inline hw02/hw02.sln:1]: Remove from PR

File: hw03/t1/Queue.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            if (IsEmpty())
+                throw new InvalidOperationException("Queue is empty");
+
+            T head = _elements[_start];
+            _elementsCount--;
+            IncrementIndex(ref _start);
+            
+            return head;
+        }
+
```
Comment (IgorKheidorov) [inline hw03/t1/Queue.cs:70]: Cool!

**Review by IgorKheidorov — APPROVED** [review]
Cool!

**Comment by Vlad311010** [comment]
merge

**Pull request merged**